### ISSUE TITLE
Load fails on local postgres because of no-ssl

### ISSUE THREAD

**Issue by gracecodes144** [infomodels/database#6]
```
$ infomodels -s chop_pedsnet -m pedsnet -v 2.2.0 --loglvl DEBUG -d postgresql://localhost/pedsnet_dcc_v22 load DevelopmentDataset
INFO[0000] beginning dataset loading                     directory=DevelopmentDataset
FATA[0001] Database Open failed                          DataModel=pedsnet DbUrl=postgresql://localhost/pedsnet_dcc_v22 ModelVersion=2.2.0 SearchPath=chop_pedsnet Service=https://data-models-service.research.chop.edu/ err=Error opening postgresql://localhost/pedsnet_dcc_v22: pq: SSL is not enabled on the server
```

This happens because the `pq` default `sslmode` is to require ssl. I think we should make this configurable at the CLI, right?

**Comment by gracecodes144** [comment]
I'm going to try just setting `sslmode` to `"disable"` all the time and see if that works for us.

**Comment by murphyke** [comment]
OK, yes I got used to specifying sslmode in the dburi, but it would be nice not to make people do that.

**Comment by murphyke** [comment]
What is "default"?  Is that "prefer"?

**Comment by gracecodes144** [comment]
No, I meant "disable". Edited it but you must have already loaded the page.

**Comment by gracecodes144** [comment]
Oh, sslmode in the URI, that makes sense too. Tried that and now I'm getting this:

```
FATA[0003] Load() failed                                 DataModel=pedsnet DbUrl=postgresql://localhost/pedsnet_dcc_v22?sslmode=disable ModelVersion=2.2.0 SearchPath=chop_pedsnet Service=https://data-models-service.research.chop.edu/ err=Error running command with `sh -c`: psql "dbname=pedsnet_dcc_v22 host=localhost sslmode=disable" -c "\COPY chop_pedsnet.drug_exposure(days_supply, dose_unit_concept_id, dose_unit_source_value, drug_concept_id, drug_exposure_end_date, drug_exposure_end_time, drug_exposure_id, drug_exposure_order_date, drug_exposure_order_time, drug_exposure_start_date, drug_exposure_start_time, drug_source_concept_id, drug_source_value, drug_type_concept_id, eff_drug_dose_source_value, effective_drug_dose, frequency, lot_number, person_id, provider_id, quantity, refills, route_concept_id, route_source_value, sig, stop_reason, visit_occurrence_id) FROM 'DevelopmentDataset/drug_exposure.csv' (FORMAT csv, HEADER true, ENCODING 'utf-8', FORCE_NULL(days_supply, dose_unit_concept_id, dose_unit_source_value, drug_concept_id, drug_exposure_end_date, drug_exposure_end_time, drug_exposure_id, drug_exposure_order_date, drug_exposure_order_time, drug_exposure_start_date, drug_exposure_start_time, drug_source_concept_id, drug_source_value, drug_type_concept_id, eff_drug_dose_source_value, effective_drug_dose, frequency, lot_number, person_id, provider_id, quantity, refills, route_concept_id, route_source_value, sig, stop_reason, visit_occurrence_id))" (STDERR: exit status 1)%!!(MISSING)(EXTRA string=ERROR:  relation "chop_pedsnet.drug_exposure" does not exist
)
```

I thought the load command would create the tables?

**Comment by gracecodes144** [comment]
For posterity, the reason the table creation was not working was that I needed to create the vocabulary first (using the "pedsnet-vocab" model), then load the core data (using the "pedsnet-core" data model and including the "vocabulary" schema in the search_path).

Closing because the problem here is a non-intuitive user interface, which is not the priority at the moment. The code is in fact functional if you know the quirks of using it.
